### PR TITLE
[TIC] Cleanup + Checkout Success mobile/accessibility refactor

### PR DESCRIPTION
**Pull request by henrykaus**
### Description
Did a general cleanup of files for future capstone teams/general consistency. Changes include:
* Fixed minor issues with readme, and removed step about needing to type in `thisisunsafe`
* Changed admin `showings` routes to `events` to more accurately represent where those routes lead to
* Renamed `ShowingActivenessToggle` to `ActivenessGroupToggle` to separate its name from the concept of showings
* Removed the unused `eventsSlice`
* Moved `ticketingSlice` to the `ticketingmanager` directory
* Removed unused functions from `ticketingSlice`
* Changed remaining `console.log`s to `console.error`s on the ticketing side of things
* Removed remaining `console.log`s from `ticketingSlice`
* Renamed `showings` folder in ticketingmanager to `Event`, as well as `ShowingsUpdated` to `components`
* Fixed capitalization of many components
* Resolved eslint issues in CRM DateRange components.
* Fixed mobile view of CRM dashboard

Made visual/accessibility/mobile improvements to the CheckoutSuccess Page

**Before:**
image
**After:**
image

### Risks
There may have been a spot where I missed renaming a route.

### Validation
I checked most areas that used the `/ticketing/showings` routes. Also no console errors in the docker client, so all imports should have been renamed properly.

### Operating System
Windows 11

### CODE REVIEW

File: client/src/components/Ticketing/ticketingmanager/ticketingSlice.ts [hunk 1]
```diff
@@ -253,40 +253,6 @@ export const fetchDiscountData = createAsyncThunk(
       minEvents: discountArray[0].min_events,
     };
 
-    console.log('Discount returned:', discountArray[0]);
```
Comment (henrykaus) [inline client/src/components/Ticketing/ticketingmanager/ticketingSlice.ts:256]: This and below is entirely for console.logging

File: client/src/components/Ticketing/ticketingmanager/Season/components/EventCard.tsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import React, {useState} from 'react';
```
Comment (henrykaus) [inline client/src/components/Ticketing/ticketingmanager/Season/components/EventCard.tsx:1]: `useState` was unused

File: client/src/components/Ticketing/ticketingmanager/ticketingSlice.ts [hunk 3]
```diff
@@ -726,62 +692,6 @@ export const selectEventData = (
   }
 };
 
-/**
```
Comment (henrykaus) [inline client/src/components/Ticketing/ticketingmanager/ticketingSlice.ts:729]: This and below were unused

**Comment by benfaustin95** [comment]
It all looks good to me, but because so many files are touched maybe one more set of eyes would be worth it.

File: README.md [hunk 4]
```diff
@@ -112,13 +111,12 @@ If that does not work, you can try `docker-compose down`, `docker-compose build
 
 This section covers the Playwright automated testing setup that has been configured for this project. Currently, the `./client/` directory is the only part with Playwright setup. The `./server/` folder will get it later once authentication issues have been resolved (Currently reworking the server tests to work without the need to connect to Auth0 as we will blow through the limit for API calls in no time as it currently does 2-5 Auth0 API requests per test and 2 times per login/page refresh).
 
-Before you begin running tests, make sure you have a TEST_EMAIL and TEST_PASSWORD set in your `.env`. Please refer to the `.env.dist` example. This will allow the `auth.setep.ts` test setup file to authenticate and save the authenticated browser context locally to be used for subsequent
-tests.
+Before you begin running tests, make sure you have a TEST_EMAIL and TEST_PASSWORD set in your `.env`. Please refer to the `.env.dist` example. This will allow the `auth.setep.ts` test setup file to authenticate and save the authenticated browser context locally to be used for subsequent tests.
```
[the remark this answers is not in the document]
Comment (henrykaus) [reply README.md]: Good suggestion, I added a little bit to this section

**Review by HaliTran — APPROVED** [review]

**Pull request merged**